dts/arm/cypress/psoc6.dtsi

```diff
@@ -0,0 +1,83 @@
+#include <cypress/mem.h>
+
+/ {
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		cpu@0 {
+			device_type = "cpu";
+			compatible = "arm,cortex-m0+";
+			reg = <0>;
+		};
+		cpu@1 {
+			device_type = "cpu";
+			compatible = "arm,cortex-m4f";
+			reg = <1>;
+		};
+	};
+
+	flash-controller@4001E000 {
+			compatible = "nrf,nrf51-flash-controller";
+			reg = <0x4001E000 0x518>;
```

Comment:
As mentioned in previous comment, these look like values for Nordic nrf5xxxx chip.